K2Bridge/KustoDAL/KustoQueryExecutor.cs

```diff
@@ -57,13 +57,26 @@ internal class KustoQueryExecutor : IQueryExecutor
         /// <returns>A new instance of <see cref="KustoConnectionStringBuilder"/>.</returns>
         public static KustoConnectionStringBuilder CreateKustoConnectionStringBuilder(IConnectionDetails connectionDetails)
         {
-            var conn = new KustoConnectionStringBuilder(
-                connectionDetails.ClusterUrl,
-                connectionDetails.DefaultDatabaseName)
-                .WithAadApplicationKeyAuthentication(
-                    connectionDetails.AadClientId,
-                    connectionDetails.AadClientSecret,
-                    connectionDetails.AadTenantId);
+            KustoConnectionStringBuilder conn;
```

Comment:
I assume that's from the other branch that slipped in?

Reply:
Good catch !
Indeed, I will remove it, sorry for that !